include/ck/tensor_operation/gpu/device/impl/device_cgemm_4gemm_xdl_cshuffle.hpp

```diff
@@ -598,10 +598,26 @@ struct DeviceCGemm_4Gemm_Xdl_CShuffle
                                  [[maybe_unused]] index_t K,
                                  [[maybe_unused]] index_t StrideA,
                                  [[maybe_unused]] index_t StrideB,
-                                 index_t StrideC) override
+                                 index_t StrideC) const override
     {
         return 2 * sizeof(CDataType) * GetCElementSpaceSize(M, N, StrideC);
     }
+
+    std::size_t GetWorkSpaceSize(const BaseArgument* base_arg) const override
```

Comment:
can we rename the other method GetWorkspaceSize() to GetWork**S**paceSize() ?

Reply:
I do not know if there's already anybody who's using it (actually low probability). But I do not want to make such changes in API. BTW Isn't `workspace` grammatically correct and better than `WorkSpace` wording?